src/doing/pr/create_pr.py

```diff
@@ -129,6 +133,7 @@ def cmd_create_pr(
     command += f'--draft "{str(draft).lower()}" '
     command += f'--work-items "{work_item_id}" '
     command += f'--source-branch "{branch_name}" '
```

Comment:
Shouldn't this also change? Also related to a previous question, about adding two options instead of just one.

Reply:
Nevermind, I got confused.